vms/proposervm/state_syncable_vm_test.go

```diff
@@ -564,12 +565,28 @@ func TestStateSummaryAcceptOlderBlock(t *testing.T) {
 
 	summary, err := vm.GetStateSummary(context.Background(), reqHeight)
 	require.NoError(err)
+	require.Equal(summary.Height(), reqHeight)
 
-	// test Accept skipped
+	// test Accept summary invokes innerVM
+	calledInnerAccept := false
 	innerSummary.AcceptF = func(context.Context) (block.StateSyncMode, error) {
+		innerVM.LastAcceptedF = func(context.Context) (ids.ID, error) {
+			return innerSummary.ID(), nil
+		}
+		innerVM.GetBlockF = func(context.Context, ids.ID) (snowman.Block, error) {
+			return innerBlk, nil
+		}
+		calledInnerAccept = true
```

Comment:
We're changing the behavior from the innerVM Accept never getting called to updating the last accepted block, so we need to update the mocked functions.

This ensures that in `SetState`, we correctly align the ProposerVM to the innerVM's updated height.